angular_test/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: angular_test
 author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/angular
 description: Testing runner and library for AngularDart
-version: 1.0.2-alpha+1
+version: 1.0.2-alpha+2-dev
```

Comment:
Please don't bump the versions yourself. We will handle this when we go to publish, based on all of the changes that are included since the last release.

Instead, just add an entry in the CHANGELOG.

Reply:
Ok.

Reply:
Done